Orange/tests/test_logistic_regression.py

```diff
@@ -14,7 +14,8 @@ def test_LogisticRegression(self):
         ca = CA(results)
         self.assertTrue(0.8 < ca < 1.0)
 
-    def test_LogisticRegressionPreprocessors(self):
+    @unittest.skip("Re-enable when Logistic regression supports normalization.")
```

Comment:
I'd add a TODO or FIXME marker. Easier to spot a TODO than no TODO (e.g. PyCharm colors TODOs differently).